File: divisor.rb
=begin
    My Name is Vincent Peterson and I 
    learned ruby by applying to app academy. I didn't go there
    though. I went to college instead.
    This program helped me with my math homework.
=end

def divisorby(n)
    output = []
    (1..n).each{|num|
        if n % num == 0
            output << num
        end 
    }
    return output 
end
puts 'Progam is running...'
print 'enter number: '
num = gets.to_i
if num > 0
    print divisorby(num) 
else
    puts 'Please enter a positive integer'
end

File: divisor.py
'''
My name is Vincent Peterson and I learned python 
from watching Youtube videos
'''
x = input('please enter a positive integer: ')
x = int(x)
array = []
output = []
i = 0
if x > 0:
    while i < x:
        i += 1
        if x % i == 0:
            output.append(i)


    print(output)
else:
    print('fuck you')
